src/utils/common_spaces.jl

```diff
@@ -100,7 +100,42 @@ function make_hybrid_spaces(
         z_mesh,
     )
     z_grid = Grids.FiniteDifferenceGrid(z_topology)
-    if isnothing(surface_warp)
+
+    if parsed_args["topography"] == "Earth"
+        @info "SpaceVaryingInputs: Remapping orography onto spectral space"
+        z_surface = SpaceVaryingInputs.SpaceVaryingInput(
+            AA.earth_orography_file_path(;
+                context = ClimaComms.context(h_space),
+            ),
+            "z",
+            h_space,
+        )
+        parent(z_surface) .=
+            ifelse.(parent(z_surface) .< FT(0), FT(0), parent(z_surface))
+        Δh_scale = Spaces.node_horizontal_length_scale(h_space)
+        Hypsography.diffuse_surface_elevation!(
+            z_surface;
+            κ = FT((Δh_scale)^2 / 100),
```

Comment:
Could you add a comment on where this coefficient comes from?

Reply:
Updated, E3SM documentation guides this (otherwise it can be estimated from the grid resolutions and some nominal CFL)